resources/codegen_inputs/ift.rs

```diff
@@ -141,35 +141,35 @@ table MappingEntries {
 }
 
 table EntryData {
-  format: EntryFormatFlags,
+  format_flags: EntryFormatFlags,
```

Comment:
was this running into some problem? It's possible we have code that treats 'format' as special or otherwise generates a 'format' token that would shadow a similarly named field..

Reply:
Just changing this to match the naming we use in the spec: https://w3c.github.io/IFT/Overview.html#mapping-entry-formatflags